### DISCUSSION TITLE
Admin panel Karavan

### DISCUSSION THREAD

**Discussion by rapetum228** [apache/camel-karavan#1068] [Q&A] [answered]
Hello. Is there any way to customize the administrator panel to work in Camel Karavan? I am interested in the possibility of entering a username and password before starting the development of integrations.

**Comment by gun2rin** [comment]
Hi.  You can use basic authentication as a minimum. 
Just set  necessary values in karavan's application properties for your environment. Dev in case below.

```
%dev.karavan.auth=public
%dev.quarkus.oidc.enabled=false
%dev.quarkus.http.auth.basic=true
%dev.quarkus.security.users.embedded.enabled=true
%dev.quarkus.http.auth.permission.authenticated.enabled=true
%dev.quarkus.http.auth.permission.public.enabled=false
%dev.quarkus.security.users.embedded.plain-text=true
%dev.quarkus.security.users.embedded.users.admin=your-password-here
```

login: admin
pass: your-password-here

I'm also sure there are other ways to authenticate. Keykloack, for example.

**Reply by rapetum228** [reply]
Thanks for the reply. I mean authorization in order to restrict free access to the page where integration development takes place. 

To be like this: the user cannot open the page for development until he/she enters login and password.

**Reply by gun2rin** [reply, marked as answer]
I mean the same. 
A user has no the access to any page until authentication.
![image](https://github.com/apache/camel-karavan/assets/16078869/07699952-1197-4e62-9844-c643f83488c2)
